### ISSUE TITLE
Minecraft: PlayStation Vita Edition [PCSB00560]

### ISSUE THREAD

**Issue by MilesTailsFan2005** [Vita3K/compatibility#1372]
# App summary
- Minecraft: PlayStation Vita Edition
- PCSB00560
- 1.84 (Latest Version) 

# Vita3K summary
- Version: v0.1.6 3057
- Build number: a601eefd
- Commit hash: https://github.com/vita3k/vita3k/commit/[commit] 
- CPU backend: Dynarmic.
                            When testing games Dynarmic should be prefered as it normally gives better results than Unicorn
                            You can change the CPU backend on the emu settings -->
- GPU backend: OpenGL (Broken In Vulkan)

                          

# Test environment summary
- Tested by: ?
- OS: Windows 10
- CPU: Intel(R) Xeon(R) CPU E5-1650 0 @ 3.20GHz   3.20 GHz
- GPU: NVIDIA GTX1050TI
- RAM: 8 GB

# Issues
Works Fine In OpenGL 
Requires Latest Version 1.84
Broken On Vulkan
Pressing There is not enough free space on the memory card it
should have a option saying ok pressing that lets you play (May Reparative Coming Up Making It Little Annoying To Play)
Little Green Flickers May Show On The Hud And Menu

# Screenshots
![Title](https://user-images.githubusercontent.com/52087744/226109833-5d31208f-d2b1-4c87-ab53-10131ecc3fe0.png)

![InGame](https://user-images.githubusercontent.com/52087744/226109840-148fb446-27b2-4627-88e8-a27aec3e9565.png)

![InGame2](https://user-images.githubusercontent.com/52087744/226109850-e4a7ae1d-1b1b-42ae-909a-b9726fd2c4a5.png)

![image](https://user-images.githubusercontent.com/52087744/226110192-70f4b295-465f-4117-9e3e-760017dc9c1a.png)

# Log

# Recommended labels

- https://github.com/Vita3K/compatibility/labels/vulkan%20issue
- https://github.com/Vita3K/compatibility/labels/Ingame%20-
- https://github.com/Vita3K/compatibility/labels/shader%20bug

**Comment by MilesTailsFan2005** [comment]
If Anyone Wants Gameplay Test Of This Here You Go 
https://youtu.be/_iZZT6ErbEE

**Comment by NickSDP10** [comment]
Que buen juego

**Comment by MilesTailsFan2005** [comment]
Yeah though it needs the latest version to look more playable its how cemu acts when you try and play minecraft

**Comment by MilesTailsFan2005** [comment]
Update The Message "Their Is Not Enough Space Free" Is Officially Gone!
I Saved A New World And Opened It Multiple Times And Haven't Gotten The Error

**Comment by HttpAnimation** [comment]
Can someone send me the download for this version of Minecraft pls

**Comment by AnasAdelAlsharkawy** [comment]
Pls can you fix the broken menu on android

**Comment by AnasAdelAlsharkawy** [comment]
> 

Nopaystation.com

**Comment by ansye** [comment]
اين التحميل اكتبها بالفصحى عشان يترجمونها

**Comment by sosksaabobusa** [comment]
How to download???

**Comment by UsualVII** [comment]
# Vita3K Build: 3502
- Version: v0.1.9
- Commit hash: https://github.com/Vita3K/Vita3K/commit/4ac8a416
- CPU backend: Dynamic

# App Info
- App name: Minecraft: Playstation Vita Edition
- App serial: PCSB00560
- App version: 1.01
- Region: EUROPE

# API & Info
- Tested VULKAN and OGL
- Audio works as expected 

# Test environment summary
- Tested by: UsualVII
- OS: Windows 11
- CPU: Intel i5-11600k
- GPU: RTX 3060 Ti 8GB
- RAM: 32GB

# Issues
- As today the game is on Black Screen (just like the JP Version)

# Notes:
- Common error: Mask not implemented in the vulkan renderer! **[VULKAN ONLY ERROR]**
- Common error 2: Open_file (sceAppUtilSaveDataSlotSearch) returned 0x80010002

# Recommended labels
- Nothing

# Logs
[vita3k.log](https://github.com/Vita3K/compatibility/files/13540939/vita3k.log)

# Screenshot
![image](https://github.com/Vita3K/compatibility/assets/88155892/f64f400a-bce6-4145-bca8-429c501e9a68)

**Comment by THEnoite** [comment]
![Screenshot_20240625-234911](https://github.com/Vita3K/compatibility/assets/173866219/c2ce1db7-21e3-4568-980e-31637e080c46)
Can someone help me. Why Minecraft is not showing an image

**Comment by THEnoite** [comment]
Can someone help me. Why Minecraft is not showing an image

**Comment by THEnoite** [comment]
Can someone help me. Why Minecraft is not showing an image

**Comment by FuriosEthan** [comment]
hopefully this gets fixed

**Comment by Slade1972** [comment]
# Vita3K summary
- Version: v0.2.0
- Build number: 3803
- Commit hash: https://github.com/vita3k/vita3k/commit/8c71bdd0
- CPU backend: Dynarmic
- GPU backend: OpenGL

# Test environment summary
- Tested by: Slade
- OS: Microsoft Windows 10 Professional (build 19045)
- CPU: Intel(R) Core(TM) i7-3770K CPU @ 3.50GHz
- GPU: NVIDIA GeForce GTX 1060 6GB
- RAM: 16 GB

# Issues
- The game requires two things to run corectly:
  - OpenGL
  - Updating to version 1.84
- The game will not run correctly if using Vulkan, even if updated.
- The game takes a while to load and can freeze during heavy loading.

# Screenshots

![Image](https://github.com/user-attachments/assets/d045e2c2-8d50-47fc-b125-0f9dc35832ab)
![Image](https://github.com/user-attachments/assets/8001eb75-4509-42c4-b01b-0225eee8766f)
![Image](https://github.com/user-attachments/assets/2cb040d7-7eaa-4829-964a-afdfd272070a)

# Log

[PCSB00560 - \[Minecraft_ PlayStation®Vita Edition\].log](https://github.com/user-attachments/files/20851836/PCSB00560.-.Minecraft_.PlayStation.Vita.Edition.log)

# Recommended labels

- Ingame +
- Vulkan issue

**Comment by HylianSoul** [comment]
I have tested [a fork of the Vita3K emulator](https://github.com/nckstwrt/Vita3K-Plus) that solves the problem with Vulkan.

<img width="951" height="566" alt="Image" src="https://github.com/user-attachments/assets/12ceb48f-54e2-49bb-ae40-23fe9025bfc6" />

I think it could be an interesting information.


## Device specs

OS: Debian GNU/Linux forky/sid (forky) x86_64
CPU: AMD Ryzen 5 3500U (8) @ 3.70 GHz
GPU: AMD Radeon Vega 8 Graphics [Integrated]
RAM: 32 GiB

The fork, for now, offers only Windows and Android builds. I executed the Windows version using ```proton-cachyos-11.0-20260703-slr-x86_64_v3```.
